reddit_get.py

```diff
@@ -25,6 +25,20 @@
 except:
     str_input = input
 
+# Python 2/3 compatibility

```

Comment:
Rather than duplicating these methods, you can just call them on gallery_get, which is already imported.

If you search this file for "gallery_get.safestr" you will see how it's using the "safestr" method from gallery_get.